src/main/python/pythonparser/pythonparser_3.py

```diff
@@ -21,23 +26,22 @@ def read_file_to_string(filename):
     return s
```

Comment:
Did you refactor this file? If the answer is yes change the function by using `open with`. But if you were not refactoring don't change it

Reply:
I didn't refactor this file, but I can rewrite function with `open with`.
Or refactor the file partially is a bad idea?

Reply:
Refactoring always is a good idea 😂 If it is not difficult it would be great!